index.js

```diff
@@ -65,9 +77,9 @@ module.exports = (number, options) => {
 		return prefix + numberString + ' ' + UNITS[0];
 	}
 
-	const exponent = Math.min(Math.floor(Math.log10(number) / 3), UNITS.length - 1);
+	const exponent = Math.min(Math.floor(Math.log(number) / Math.log(options.binary ? 1024 : 1000)), UNITS.length - 1);
 	// eslint-disable-next-line unicorn/prefer-exponentiation-operator
-	number = Number((number / Math.pow(1000, exponent)).toPrecision(3));
+	number = Number((number / Math.pow(options.binary ? 1024 : 1000, exponent)).toPrecision(3));
```

Comment:
`options.binary ? 1024 : 1000` is used twice. Could be a constant.

Reply:
If I change it to use `Math.log10`  then it is only used once.